doc/release-notes.md

```diff
@@ -1,5 +1,16 @@
 # release notes
 ## current release
+- release date: 2021-05-21
+- status: available
+- changes:
+  - Added files below:
```

Comment:
```suggestion
  - Released files below:
```

I realized that we have not been documenting release changes as closely as we perhaps should be, since we did not have a ticketing system up and running.

The files downloaded OK for me, but we should also add the changes that were made (MAFs which were updated due to header issue, ref ticket) and counts file added (ticket).